--- Makefile.orig	2017-07-24 06:57:48.196985820 +0000
+++ Makefile	2017-07-24 07:04:36.984992198 +0000
@@ -1079,13 +1079,15 @@
 # containing $src, and passes TESTS_ENVIRONMENT.
 am__check_pre = \
 $(SH_E_WORKAROUND);				\
+$(am__vpath_adj_setup) $(am__vpath_adj)		\
 tst=`echo "$$src" | sed 's|^.*/||'`;		\
 rm -f $@-t;					\
 $(mkdir_p) "$$(echo '$@'|$(_dirname))" || exit;	\
 if test -f "./$$src"; then dir=./;		\
 elif test -f "$$src"; then dir=;		\
 else dir="$(srcdir)/"; fi;			\
-$(TESTS_ENVIRONMENT) $(SHELL)
+tst=$$dir$$f;					\
+$(TESTS_ENVIRONMENT)
 
 
 # To be appended to the command running the test.  Handles the stdout
